dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+DIR="/usr/local/etc/namedb/raw/"
+
+if [ ! -d $DIR ]; then
+
+  mkdir $DIR
+  chown bind:bind $DIR
+  chmod -R 755 $DIR
+
+fi
+
+cd $DIR
+
+# Download all lists:
+
+# EasyList:
+fetch -q https://justdomains.github.io/blocklists/lists/easylist-justdomains.txt -o easylist-raw
```

Comment:
for later: we should only download what we use

Reply:
use a FETCH macro like here https://github.com/opnsense/core/blob/master/src/opnsense/scripts/firmware/changelog.sh#L32

and set full path like here https://github.com/opnsense/core/blob/master/src/opnsense/scripts/firmware/changelog.sh#L51